<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PODesignerDockingTabsResults</class>
 <widget class="QWidget" name="PODesignerDockingTabsResults">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>479</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Docking panel</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/StandardWorkPanels/build.png</normaloff>:/images/StandardWorkPanels/build.png</iconset>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"> QPushButton {
     border: 1px solid #dadbde;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
     border-radius: 2px;
     min-width: 34px;
     min-height: 24px;
 }

 QPushButton:checked {
     border: 2px solid #777777;
 }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="WidgetTabOfResults" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>380</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="layoutTabResults">
      <item>
       <widget class="QTreeView" name="posesOfResultsTV">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutControls">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutNumOfPoses">
          <property name="topMargin">
           <number>6</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string>Number of poses:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="snapshotSB">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>/</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="numOfSnapshotsSB">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutSliderButtons">
          <property name="topMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QToolButton" name="prevSnapshotPB">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>26</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/images/DockingProject/12.trajectory/PrevSnapshot.svg</normaloff>:/images/DockingProject/12.trajectory/PrevSnapshot.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>36</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="snapshotSL">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>16</height>
             </size>
            </property>
            <property name="tracking">
             <bool>true</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="nextSnapshotPB">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>26</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/images/DockingProject/12.trajectory/NextSnapshot.svg</normaloff>:/images/DockingProject/12.trajectory/NextSnapshot.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>36</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutLineSplit">
        <property name="topMargin">
         <number>24</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
